pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -19,7 +19,7 @@ title: "Professional Development Workshop Survey"
   teacher_id = enrollment[:user_id]
   teacher = DASHBOARD_DB[:users].where(id: teacher_id).first
   name = teacher.nil? ? "#{enrollment[:first_name]} #{enrollment[:last_name]}" : teacher[:name]
-  email = teacher.nil? ? enrollment[:email] : teacher[:email]
+  email = teacher.try(:[], :email).presence || enrollment[:email]
```

Comment:
Not sure why you need `.presence` - the try will evaluate to nil if email doesn't exist and `nil || enrollment[:email]` gets you the latter